concepts/numbers/introduction.md

```diff
@@ -0,0 +1,66 @@
+# Haskell Built-in Numeric Types
```

Comment:
There is quite some content in this file. While the `about.md` document can be quite elaborate, the `introduction.md` document should be quite succinct. See [the docs](https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-introduction-md) which have these rules (amongst others):

> The information provided should give the student just enough context to figure out the solution themselves.
>
> Only information that is needed to understand the fundamentals of the concept and solve the exercise should be provided. Extra information should be left for the concept's about.md document.